x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/ml/text_expansion_search.yml

```diff
@@ -245,7 +245,7 @@ setup:
               ml.tokens:
                 tokens: [{"the": 1.0}, {"comforter":1.0}, {"smells":1.0}, {"bad": 1.0}]
                 pruning_config: {}
-  - match: { hits.total.value: 5 }
+  - match: { hits.total.value: 7 }
```

Comment:
Can you explain why this hit count increased?

Reply:
This is a typo. I corrected this. Thank you.